### ISSUE TITLE
Error due to case sensitivity of filenames on Linux system

### ISSUE THREAD

**Issue by qdread** [USEPA/useeior#1]
Hi,

I experienced an issue on a Linux system. The file names are case sensitive so calling

`buildEEIOmodel(modelname = 'USEEIOv2.0-GHG')` 

returns the following error:

```
Error in utils::read.table(system.file("extdata", "USEEIO_LCIA_factors.csv",  : 
  no lines available in input
```

As you can see this is because of the case sensitivity of the file name. The function is searching for `USEEIO_LCIA_factors.csv` with lowercase f, but the file has a capital F. See below:

```
> system.file('extdata', 'USEEIO_LCIA_Factors.csv', package='useeior')
[1] "/research-home/qread/R/x86_64-pc-linux-gnu-library/3.6/useeior/extdata/USEEIO_LCIA_Factors.csv"
> system.file('extdata', 'USEEIO_LCIA_factors.csv', package='useeior')
[1] ""
```

I think it is probably necessary to go through all calls to `system.file()` in the code base and see whether the cases match the files being called. Otherwise the code will not work on a system where file names are case sensitive.

Thanks for all your work on this!

**Comment by WesIngwersen** [comment]
@qdread Thanks so much - we'll get on it!

**Comment by qdread** [comment]
I am pretty sure the only script affected is LCIAFunctions.R

**Comment by qdread** [comment]
I think I fixed the problem, see #2 .
